Doc/whatsnew/3.8.rst

```diff
@@ -149,6 +149,22 @@ extensions compiled in release mode and for C extensions compiled with the
 stable ABI.
 (Contributed by Victor Stinner in :issue:`36722`.)
 
+To embed Python into an application, a new ``--embed`` option must be passed to
+``python3-config --libs --embed`` to get ``-lpython3.8`` (link the application
+to libpython). To support Python 3.7 and older, try ``python3-config --libs``
+(without ``--embed``) on failure.
```

Comment:
I'd rather say:

To support both 3.8 and older, try `python3-config --libs --embed` first and fallback to `python3-config --libs` (without `--embed`) if the previous command fails.

Reply:
I picked your sentence, thanks.